src/Workspaces/Core/Portable/Workspace/Workspace_Events.cs

```diff
@@ -3,6 +3,7 @@
 using System;
 using System.Threading.Tasks;
 using Roslyn.Utilities;
+using Microsoft.CodeAnalysis.Text;
```

Comment:
Why does "Place 'System' directives first when sorting usings" keep getting unchecked for me, causing these problems? Argh.